udf-compiler/src/main/scala/com/nvidia/spark/udf/CFG.scala

```diff
@@ -327,4 +366,13 @@ object CFG {
         successor + (src -> dst))
     }
   }
+
+  private def isReturn(opcode: Int): Boolean = {
```

Comment:
I think you can use https://www.javassist.org/html/javassist/bytecode/analysis/Util.html#isReturn(int).
Can you check if it does the same thing you are trying to do here?

Reply:
it seems to be the case based on https://github.com/jboss-javassist/javassist/blob/master/src/main/javassist/bytecode/analysis/Util.java#L45-L47